packages/web-react/src/components/TextArea/index.tsx

```diff
@@ -0,0 +1,2 @@
+export * from './TextArea';
+export { default as TextArea } from './TextArea';
```

Comment:
It is not necessary to have this file with `.tsx`. 
If you don't use `React` or `JSX` elements in file, just have the `.ts` extension.

Reply:
Good point, but in this package we have every for every component `index` file in `.tsx`. I think this change is candidate for separate issue and PR. cc @literat